### DISCUSSION TITLE
Suddenly facing Error in initialisation command npx create-turbo@latest

### DISCUSSION THREAD

**Discussion by Lavkush3411** [vercel/turborepo#9009] [Help] [answered]
### Summary

Hello Everyone,
I was setting the monorepo using turborepo, then due to some misconfig I have deleted the entire folder and now when i am running npx create-turbo@latest then it's giving error like : " no "package.json" found at C:\Users\folder".
tried with new folder /empty folder/ system restart/ tried with yarn as well but no luck.

Attaching screenshot of same hereby, your help is appreaciated thanks in advance.

![image](https://github.com/user-attachments/assets/22a81254-553d-4650-8c9a-5e85a8169ae1)

### Example

Error in initalisation command npx create-turbo@latest.

**Comment by akash942** [comment]
facing similar issue, let me know if you were able to resolve it

**Comment by anthonyshew** [comment, marked as answer]
This was a bug that is fixed now! Sorry for the disruption!
